[Usability] mandates: search by reference, add group_by, add seq type in tree view

alexis-via opened the pull request.

Commit:
[Usability] mandates: search by reference, add group_by, add seq type in tree view

StefanRijnhart commented on account_banking_mandate/views/account_banking_mandate_view.xml:
```diff
@@ -68,11 +68,15 @@
 		<field name="model">account.banking.mandate</field>
 		<field name="arch" type="xml">
 			<search string="Search Banking Mandates">
```
Thanks for the improvements! Odd spacing here, as in the other XML. Are tabs involved, in the original or new code? Maybe this is a good time to reformat on indents of 4 spaces?

alexis-via replied:
yes, it's caused by the fact that there are tabs everywhere in the XML file !!! And the lines I added have spaces.

pedrobaeza commented:
Can you please add also by type of mandate (CORE, B2B)?

alexis-via commented:
@pedrobaeza yes, i'll update the PR with your remark when I have 5 min

Commit:
Add scheme in mandate tree+search view
Replace tabs by spaces in mandate views
account_banking_mandate/views/account_banking_mandate_view.xml: convert from dos to unix format

alexis-via commented:
This PR is finished, we can merge it. Apart from the replacement of tabs by spaces, it's just a few more lines in tree+search view of mandates. No risk.

pedrobaeza commented:
All is correct as said by Alexis, so I merge. :+1:
